s29741-pj-JakubKolad/30.09/Z1.txt

```diff
@@ -0,0 +1,50 @@
+1.Uruchom terminal.ctrl+alt+t
+
+2.Wyświetl zawartość aktualnego katalogu.ls
+
+3.Sprawdź gdzie aktualnie się znajdujesz.pwd
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej.cd Desktop/
+
+5.Sprawdź gdzie jesteś.pwd
+
+6.Wyświetl zawartość bieżącego katalogu.ls
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z odpowiednim parametrem).ls /home
+
+8.Przejdź do katalogu Obrazy (lub innego katalogu, który jest dostępny - podaj wprost, że zmieniasz katalog na inny bo np. Obrazy nie są u Ciebie dostępne) w twoim katalogu domowym za pomocą ścieżki względnej. cd Pictures/
+
+9.Sprawdź gdzie jesteś.pwd
+
+10.Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././. Katalog nie uległ zmianie.
+
+*	W opisie rozwiązania proszę umieścić również to jak rozumiesz co się właściwie stało.	Dlaczego znalazła/eś się w tym miejscu do którego przeniosła Cię ta ścieżka?
+
+Rozumiem to jako polecenie "zostań tu gdzie jesteś". Po doczytaniu w Internecie rozumiem, że "./" pozwala np. na edytowanie plików "w dole" ścieżki,
```

Comment:
Nie. Nie "zostań tu gdzie jesteś" tylko aktualny katalog. To lokalizacja "tutaj gdzie jestem teraz". To impliku to co napisałeś ale "./" nie jest to związane z tym by edytować pliki w podkatalogach. To dodatkowy efekt uboczny. Celowy. Ale mimo wszystko tylko efekt uboczny. Poza tym można wskazywać lokalizacje relatywne bez potrzeby podawania "./" na początku czyli tak jak zrobiłes np w zadaniu 8. Tu ie podałeś przecież `cd ./Pictures/` tylko `cd Pictures/`.